### ISSUE TITLE
Bug - Options Menu - docs failing a11y testing

### ISSUE THREAD

**Issue by nicolethoen** [patternfly/patternfly#5805]
It's disrupting PR builds / CI

```
147/265   /components/menus/options-menu
violations: aria-required-children, aria-required-parent, list, listitem
```

Seems to be related to changes introduced in https://github.com/patternfly/patternfly/pull/5753
and revealed once a11y tests were cleaned up in https://github.com/patternfly/patternfly/pull/5705
